### DISCUSSION TITLE
Link to forked repository

### DISCUSSION THREAD

**Discussion by fgreg** [community/community#21920] [New to GitHub]
<p>I don’t know how to request a feature to be added to GitHub so thought I’d post it here:</p>
<p>I’d like a link that is the reverse of the “forked from: author/repo” when you are looking at your repository that was forked from some upstream project.</p>
<p>When I view the upstream project, I’d like a link that shows my forks of that project.</p>
<p>Described in more detail here: <a href="https://stackoverflow.com/questions/48775748/easy-way-to-get-to-my-fork-when-i-look-at-the-repository-i-forked-from/48775749#48775749" rel="nofollow noopener">https://stackoverflow.com/questions/48775748/easy-way-to-get-to-my-fork-when-i-look-at-the-repository-i-forked-from/48775749#48775749</a></p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/fgreg">@fgreg</a>,</p>
<p>Thanks for being part of the GitHub Community Forum! I’ll help as best I can.<br>
You actually already can find your forks of a repo in the way you describe. If you go to the repo’s main page, in the upper right corner, next to “Watch” and “Star”, you’ll see “Fork”. If you click on the number there, that will take you to a page where you can see all the forks of that repository, listed by username.</p>
<p><img src="https://user-images.githubusercontent.com/16922726/181067529-39e1d577-e8be-4de6-87fa-c3c9d46282e8.png" alt="Repo's forks.png" data-base62-sha1="1ysY7onIGeVzeqRJqU4eSH3LOc8" width="502" height="61"></p>
<p>If that’s not quite what you’re looking for and you’d still like to submit a feature request, for now, the best way to submit bug reports or feature requests is via our contact page (<a href="http://github.com/contact" rel="nofollow noopener">github.com/contact</a>). GitHub Staff will see and respond to those requests there. You can include a link to this forum post in your message so that you don’t have to write out the entirety of your idea again.</p>
<p>Cheers!</p>

**Comment by fgreg** [comment]
<p>Thanks for the reply. Yes that is what I usually do, but it would be great for it to be within one click.</p>
<p>When the network graph is large it can take some time to load and you have to find yourself in the list of all forks of the repo. All-in-all just a little annoying; would be sweet if it just told me right away that I have a fork to navigate to. I did submit a request using the contact link.</p>
